source/sync/data-model/update-data-model/update-data-model-production.txt

```diff
@@ -0,0 +1,431 @@
+.. _update-data-model-production:
+
+=================================
+Update a Data Model in Production
+=================================
+
+.. meta::
+   :description: Update your data model for an app in production.
+   :keywords: Realm, object model, C++ SDK, Flutter SDK, Java SDK, Kotlin SDK, .NET SDK, Node.js SDK, React Native SDK, Swift SDK
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+After your app goes to production, you might need to make additional updates to
+you data model. Production-level apps can take advantage of schema versioning,
+which supports backward-compatibility with your existing schemas
```

Comment:
might also worth be stating that schema versions are also forward-compatible (compatible with any future schema version)